src/pages/NewChairConsole.js

```diff
@@ -106,7 +106,7 @@ const Records = (props) => {
     if (!isNil(e)) {
       switch (e.status) {
         case 'Open' : {
-          const votes = filter({type: 'DAC', dacUserId: currentUserId})(electionInfo.votes);
+          const votes = filter({type: 'DAC', dacUserId: currentUserId, electionId: e.electionId})(electionInfo.votes);
           const isFinal = !isEmpty(votes.find((voteData) => !isNil(voteData.vote)));
```

Comment:
What is the purpose of displaying Final instead of Vote when both buttons do the exact same thing? (talking about the isFinal check on the label in line 115)

Reply:
Talked with Greg yesterday, 'Final" is a legacy feature so the UI should only show "Vote". The correction was made in DUOS-1084 (since it affects the Vote Button), so once that's merged in the changes should be seen in this PR.